src/core/EdFi.DataManagementService.Core/CoreFacade.cs

```diff
@@ -147,4 +148,36 @@ public async Task<FrontendResponse> DeleteById(FrontendRequest frontendRequest)
         await _deleteByIdSteps.Value.Run(pipelineContext);
         return pipelineContext.FrontendResponse;
     }
+
+    /// <summary>
+    /// DMS entry point for data model information from ApiSchema.json
+    /// </summary>
+    public IList<DataModelInfo> GetDataModelInfo()
```

Comment:
Move this out of AspNetCore.  More of the metadata endpoints could be moved out, but this is the minimal amount to get clean separation.